tests/globalhelper/runhelper.go

```diff
@@ -47,5 +48,44 @@ func LaunchTests(testSuites []string, skipRegEx string) error {
 	cmd.Args = append(cmd.Args, testArgs...)
 	cmd.Dir = Configuration.General.TnfRepoPath
 
+	if os.Getenv("DEBUG_TNF") == "true" {
+		os.Setenv("LOG_LEVEL", "trace")
+
+		buildNumber, present := os.LookupEnv("BUILD_NUMBER")
+
+		var debugFolderName string
+
+		if present {
+			debugFolderName = "Debug-" + buildNumber
+		} else {
+			debugFolderName = "Debug-Manual"
+
+		}
+
+		folderPath := filepath.Join(Configuration.General.ReportDirAbsPath, debugFolderName, tcNameForFolder)
```

Comment:
Let's build following struct Configuration.General.ReportDirAbsPath/Test_suite_name_folder/Test_case_name_folder.

After every run we will overwrite Configuration.General.ReportDirAbsPath/**Test_suite_name_folder**

Reply:
Do you mean in the AfterSuite?

again, but then we lost the logs, we need them to see what went ok/ wrong.

Reply:
done, added deletion in run-tests.sh script.